src/schedule/schedule.c

```diff
@@ -52,6 +52,10 @@ static void scheduler_register(struct schedule_data *scheduler)
 		/* init schedulers list */
 		*sch = rzalloc(SOF_MEM_FLAG_KERNEL,
 			       sizeof(**sch));
+		if (!*sch) {
+			tr_err(&sch_tr, "scheduler_register(): allocation failed");
+			return;
```

Comment:
I suppose if we cannot even allocate a list of schedulers, that's a rather certain reason to panic, but not too important, it will die later anyway

Reply:
actually better to panic here today. updated.